packages/code-du-travail-data/dataset/fiches_ministere_travail/package.json

```diff
@@ -8,6 +8,7 @@
     "url": "https://github.com/SocialGouv/code-du-travail-numerique/issues"
   },
   "dependencies": {
+    "google-spreadsheets": "^2.0.0",
```

Comment:
No `yarn.lock` changes ? :thinking:

Reply:
this package is not handle by lerna

Reply:
my bad; the package is handle by lerna, but yarn not update yarn.lock so I guess its ok